src/ripple/app/ledger/ConsensusTransSetSF.cpp

```diff
@@ -57,10 +58,10 @@ void ConsensusTransSetSF::gotNode (
             SerialIter sit (s.slice());
             STTx::pointer stx = std::make_shared<STTx> (std::ref (sit));
             assert (stx->getTransactionID () == nodeHash);
-            getApp().getJobQueue ().addJob (
+            app_.getJobQueue ().addJob (
                 jtTRANSACTION, "TXS->TXN",
-                [stx] (Job&) {
-                    getApp().getOPs().submitTransaction(stx);
+                [&, stx] (Job&) {
+                    app_.getOPs().submitTransaction(stx);
```

Comment:
I just fired up an instance of this code and ran it as a server.  Two time now I've hit a segfault 11 on this line.  I don't think we're ready for prime time yet.